proto/stored_invocation.proto

```diff
@@ -17,3 +17,10 @@ message StoredInvocation {
   int64 invocation_status = 10;
   bool success = 11;
 }
+
+// Proto representation of the InvocationExecution. Only used in backends.
+message StoredInvocationLink {
+  string invocation_id = 1;
+  string execution_id = 2;
+  int32 type = 3;
```

Comment:
Can you make this an enum? It would be good if we could share the enum between this code and the mysql code.

Reply:
Done